ui/src/app/applications/components/application-sync-panel/application-sync-panel.tsx

```diff
@@ -172,9 +172,27 @@ export const ApplicationSyncPanel = ({application, selectedResource, hide}: {app
                                                 .filter(item => !item.hook)
                                                 .map((item, i) => {
                                                     const resKey = nodeKey(item);
+                                                    const contentStart = resKey.substr(0, Math.floor(resKey.length / 2));
+                                                    let contentEnd = resKey.substr(-Math.floor(resKey.length / 2));
+                                                    // Fix Latin letters being rendered left to right on ellipsis with direction set to rtl
```

Comment:
Updating this comment as `// Fix Latin letters being rendered left to right on ellipsis with direction set to rtl as special characters are handled as Latin letters which are naturally left to right`.

Reply:
I'm not sure I'd write it like this...

I was thinking more like:
```
// We want the ellipsis to be in the middle of our text, so we use RTL layout to put it there.
// Unfortunately, strong LTR characters get jumbled around, so make sure that the last character isn't strong.
```

For reference, `strong` is a technical term:
https://en.wikipedia.org/wiki/Bidirectional_text#Strong_characters